### ISSUE TITLE
Invention should notate whether an item is placed in inventory or on the ground

### ISSUE THREAD

**Issue by BashfulRay** [Emperorlou/Initium-ODP#849]
me again....

notating whether the result is thrown on the ground or placed in your inventory would be nice when inventing.

Yup, after thinking about it probably should have realized the kiln might be on the ground, but something notating this would be really nice, especially as inventions evolve to include many different things that may get thrown on the ground like the kiln or dug firepit

**Comment by Emperorlou** [comment]
Agreed. In addition to that, thinks like kilns and firepits and buildings will need to be build in a particular square in the 2D view. Also, "the ground" in a location is probably going to be replaced by the 2D view.
